domain-models-interface-extensions/src/main/java/org/folio/rest/tools/GenerateRunner.java

```diff
@@ -175,4 +176,33 @@ public void generate(String inputDirectory) throws Exception {
     log.info("processed: " + numMatches + " raml files");
   }
 
+  public static void createLookupList(File directory, String name, String...suffixes) throws IOException {
+    File listFile = new File(directory.getAbsolutePath() + File.separator + name);
+    Path path = Paths.get(directory.getAbsolutePath(), name);
+    try (BufferedWriter bw = Files.newBufferedWriter(path)) {
+      for (File file: directory.listFiles((FileFilter) new SuffixFileFilter(suffixes, IOCase.INSENSITIVE))) {
+        log.info("lookup entry: " + file.getName());
+        bw.write(file.getName());
+        bw.newLine();
```

Comment:
This is \r\n under Windows. Is it intended that operating system of the build server influences the line ending of the .list files in the jar?

Reply:
https://docs.oracle.com/javase/8/docs/api/java/io/BufferedWriter.html#newLine--

As long as we are aware of the platform dependent line separator, I think this is appropriate.

Reply:
We use IOUtils to read the file, which uses a BufferedReader.

https://commons.apache.org/proper/commons-io/javadocs/api-2.5/src-html/org/apache/commons/io/IOUtils.html#line.1288

Reply:
Java Files readAllLines is also safe.

https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#readAllLines-java.nio.file.Path-java.nio.charset.Charset-